src/events/request/RequestEventHandler.ts

```diff
@@ -58,6 +60,33 @@ export default class RequestEventHandler implements EventHandler<'message'> {
 			return;
 		}
 
+		if ( BotConfig.request.invalidRequestJql ) {
+			const searchResults = await this.jira.search.search ( {
+				jql: BotConfig.request.invalidRequestJql,
+				fields: ['key'],
+			} );
```

Comment:
How about changing the JQL to `` `(${ BotConfig.request.invalidRequestJql }) AND key in (${ tickets.join( ',' ) })``` and check if the result is empty so that we are not querying thousands of tickets everytime?

Reply:
Hehe good point